### DISCUSSION TITLE
How do I make animation trigger when I scroll down to that element

### DISCUSSION THREAD

**Discussion by blue-foot-xy** [animate-css/animate.css#1496] [Q&A] [answered]
Hi! I wanted to know if there is a way to trigger animation when you scroll down to the elements at the bottom. 

Right now animations are only being triggered when I refresh the page, and the animations are only being shown at the top portion that loads in the beginning, while the remaining bottom portions are just left out unanimated.

**Comment by warengonzaga** [comment]
hey @blue-foot-xy sorry but you have to use JavaScript to achieve this.
You can try this instead: https://michalsnik.github.io/aos/

**Comment by KodingKhurram** [comment, marked as answer]
This has been answered [here](https://github.com/animate-css/animate.css/discussions/1453)

**Comment by Arhell** [comment]
Try https://wowjs.uk/ 
https://michalsnik.github.io/aos/
https://webdesign.tutsplus.com/tutorials/animate-on-scroll-with-javascript--cms-36671
https://alvarotrigo.com/blog/css-animations-scroll/
https://cssanimation.rocks/scroll-animations/
